src/apps/concepts/__test__/api.test.ts

```diff
@@ -0,0 +1,75 @@
+import {authenticatedInstance, unAuthenticatedInstance} from "../../../api";
+import {APIConcept} from "../types";
+import api from "../api";
+
+const testConcept: APIConcept = {
+    display_name: "testConcept",
+    url: "url",
+    version_url: "versionUrl",
+    mappings: [],
+    retired: false,
+    source_url: "sourceUrl",
+    id: "testId",
+    external_id: "extId",
+    concept_class: "",
+    datatype: "TEXT",
+    names: [],
+    descriptions: [],
+    extras: {}
+};
+jest.mock("../../../api", () => ({
+    authenticatedInstance: {
+        put: jest.fn(),
+        get: jest.fn(() => ({activeConcepts: {
+                display_name: "testConcept",
+                url: "url",
+                version_url: "versionUrl",
+                mappings: [],
+                retired: false,
+                source_url: "sourceUrl",
+                id: "testId",
+                external_id: "extId",
+                concept_class: "",
+                datatype: "TEXT",
+                names: [],
+                descriptions: [],
+                extras: {}
+            }
+        })),
+        post: jest.fn(),
+    },
+    unAuthenticatedInstance: {
+        get: jest.fn(() => ({activeConcepts: []})),
+    }
+}));
+
+describe('api', () => {
+
+   it('should make aunthenticated get call with given url and params to get active concepts', async() => {
+       const conceptsUrl: string = "/user/concepts/";
+       const page: number = 1;
+       const limit: number = 1;
+       const q: string = "";
+       const sortDirection: string = "sortAsc";
+       const sortBy: string = "bestMatch";
+       const dataTypeFilters: string[] = [];
+       const classFilters: string[] = [];
+       const sourceFilters: string[] = [];
+       const response = await api.concepts.retrieveActive(conceptsUrl, page, limit, q, sortDirection, sortBy, dataTypeFilters, classFilters, sourceFilters);
+       expect(authenticatedInstance.get).toHaveBeenCalledWith(conceptsUrl, {
+           "params": {
+               "limit": 1,
+               "page": 1,
+               "q": "*",
+               "sortAsc": "bestMatch",
+               "timestamp": expect.anything()
+           }
+       });
+       expect(response).toStrictEqual({activeConcepts: testConcept});
```

Comment:
@tarunkumar-tw can we have only one assertion per test

Reply:
@rakesh-1010 We are checking if get call is made with correct parameters and the returned response matches the expected response. Is it okay to keep both the excepts together as they are inter-linked or do you want us to split this into two